.github/workflows/UI tests.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Instrumentation Tests
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ develop ]
```

Comment:
She'll we add master as well? Sometimes after huge merges things can get messy and unintended bugs appear.

Reply:
Can a separate issue be created for this? Because even the build test only runs for develop.

Reply:
> Can a separate issue be created for this? Because even the build test only runs for develop.

@annabauza

Reply:
No because, this issue does not specify the branch. So technically it does not fix the issue completely.

Reply:
No because, this issue does not specify the branch. So technically it does not fix the issue completely.

Reply:
Thankyou for pointing that out. Should I also create an issue to add the master branch in the build test?